mssql_go19.go

```diff
@@ -196,6 +196,7 @@ func (s *Stmt) makeParamExtra(val driver.Value) (res param, err error) {
 		}
 		schema, name, errGetName := getSchemeAndName(val.TypeName)
 		if errGetName != nil {
+			err = errGetName
```

Comment:
>	err = errGetName [](http://example.com/codeflow?start=2&length=17)

i don't think this error is meant to be fatal. It just means there's no information for the UdtInfo and TypeId fields.

Can you provide a test that shows what this fixes?